Dockerfile-chatd

```diff
@@ -1,4 +1,4 @@
 FROM wazoplatform/wazo-chatd
 
-RUN true \
-    && apt-get update && apt-get install -y netcat
+RUN apt-get update && apt-get install -y netcat git
+RUN git clone https://github.com/wazo-platform/wazo-chatd.git /usr/src/wazo-chatd
```

Comment:
Suggestion: use shallow clone (depth = 1) to reduce bandwidth usage, or get the tarball from Github to avoid installing `git` and its bazillion dependencies.

Reply:
Oh I forgot this one, thank you